### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by Anikor**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+local
```
Comment (IvanCeban) [inline .gitignore:1]: To ignore a folder with all it's content you need to add a slash after the folder name. 

**local/**

Comment (Anikor) [reply .gitignore]: fixed

Comment (IvanCeban) [reply .gitignore]: Good, but as I see you didn't create the local folder and local.txt inside it as well as the temp.txt in the root. 
You're ignoring them in gitignore, but they don't exist at all

Comment (Anikor) [reply .gitignore]: Both files are in project folder and in .gitignore

Comment (Anikor) [reply .gitignore]: https://imgur.com/a/7CeMwq9

Comment (IvanCeban) [reply .gitignore]: My bad :) 
Since they are in gitignore, they don't appear in git and are not pushed to the repo. Sorry for the confusion

**Pull request merged**